packages/protocol/thread/pause.ts

```diff
@@ -24,7 +24,7 @@ import { NodeFront } from "./node";
 import { RuleFront } from "./rule";
 import { StyleFront } from "./style";
 import { StyleSheetFront } from "./styleSheet";
-import type { ThreadFront as ThreadFrontType } from "./thread";
+import { ThreadFront as ThreadFrontType } from "./thread";
```

Comment:
:hand: technically this ought to stay as an `import type` to avoid the actual circular import if possible.

Reply:
Ahhh, fair. My Next app was saying it didn't know how to handle that type (heh) of import but in the name of keeping this PR focused, I'll remove that change.